website/content/api-docs/auth/kubernetes.mdx

```diff
@@ -25,6 +25,16 @@ access the Kubernetes API.
 | :----- | :------------------------ |
 | `POST` | `/auth/kubernetes/config` |
 
+### Deprecations
+
+The following parameters have been deprecated:
+
+* `disable_iss_validation` `(bool: true)` Disable JWT issuer validation. Allows to skip ISS validation.
+   *TODO* instead do this.
+
+* `issuer` `(string: "")` Optional JWT issuer. If no issuer is specified, then this plugin will
+   *TODO* instead do this..
```

Comment:
You may want state that the configuration will be removed in a future release of Vault, and should no longer be configured etc.